docs/archives/backend.md

```diff
@@ -0,0 +1,44 @@
+
+## Responding to Frontend Fetching Article Text
+There unfortunately isn't a single canonical source for article text on archives. There are two places that article text is stored. The first is on github. You can find the text for all the archived articles [here](https://github.com/thestanforddaily/archives-text). It's a massive repo and working with it is a pain (git doesn't scale super well, so actions like `git status` take > 10 min; it took me ~2 hours to submit corrections to all articles in the repo. Also, commit directly to master when working with archives-text). The other source for article text is in cloudsearch, as there is an article-text field in cloudsearch. Technically, the data in cloudsearch was derived from the archives-text repo, but they aren't connected in any other way; if you make changes to archive-text, they won't automatically sync to cloudsearch. There are many things to be improved here.
+
+When fetching article text on the frontend (e.g. when you go to an article and see the corresponding text to that article in the side bar), this text comes from the archives-text repository. The backend is really all handled by github and their API. The only thing that concerns us here is ensuring that this repo's data is up to date.
+
+## Responding to Frontend Fetching Issue Images
+Issue images are fetched on the frontend by openseadragon. These images live in AWS s3 buckets, so the images are fetched from there. I don't know much about this tbh so I can't really document it well
+
+## Responding to Frontend Submitting Corrections to Articles
+The frontend corrections form is powered by google forms. We're using an embedded form. Checkout [this](https://trevorfox.com/2015/06/dynamically-pre-fill-google-forms-with-mailchimp-merge-tags/) for more details
+
+## Cloudsearch
+Cloudsearch is a search engine service by AWS which powers the archives search functionality. Cloudsearch is powered by Apache Solr. 
+
+To "see" cloudsearch, go to `aws.amazon.com` and sign in. Then, in the find services search bar, search for cloudsearch. Once there, there should be a `archives-text-cloudsearch` domain. If you click on it, you can interact with the cloudsearch through aws's interface. Here you can search, and you can upload data. Note that you don't need to use AWS's own interface to do these two things. Amazon provides APIs which we can do these things programmatically. 
+
+We interact with cloudsearch programmatically in two ways:
+
+- Searching
+- Uploading Data
+
+### Searching
+Searching is pretty simple. You just submit a query to an endpoint and the endpoint responds with the search results. 
```

Comment:
Can you add a screenshot to each of these subsections? The convention I've been using is to create an `img` directory and add screenshots there.

You can then add an image with the syntax `![alt text](path to img)`. For example, `![search](img/search.png)`